task/golangci-lint/0.1/README.md

```diff
@@ -0,0 +1,53 @@
+# Unit Tests for Go Code
+
+The golangci-lint Task runs a Golang linter.
+The task is tailored to be triggered by Prow as a ProwJob.
+It uses a golangci-lint tool.
+The GolangCI tool config is read from the `.golangci.yaml` file located in a root of repository hosting tested code.
```

Comment:
There's inconsistency in tool naming. You should refer this tool as it's official name `golangci-lint`, not `GolangCI`